README.md

```diff
@@ -793,6 +795,22 @@ source ~/.bash_profile
 source ~/.bash_profile
 ```
 
+Install MongoDB C++ driver:
+
+```bash
+cd ~
+curl -LO https://github.com/mongodb/mongo-c-driver/releases/download/1.9.3/mongo-c-driver-1.9.3.tar.gz
```

Comment:
Is version 1.9.3 the actual minimum now?  If so, please remove or protect with a conditional if available on a later distribution the line: `set(CPACK_DEBIAN_PACKAGE_DEPENDS "libmongoc-1.0-0 (>= 1.3)" PARENT_SCOPE)` in mongo_db_plugin's CMakeLists.txt.

Ubuntu 16 includes a packaged version of 1.3.  Ubuntu 17 includes a packaged version of 1.7.  Ubuntu 18 includes a packaged version of 1.9.2 (at the moment).

Reply:
mongo-c-driver-1.9.3 is the C driver version recommended when compiling mongo-cxx-driver from git clone https://github.com/mongodb/mongo-cxx-driver.git --branch releases/stable --depth 1. I suppose we will need to keep an eye on their stable release and dependency versions.